waimak-core/src/main/scala/com/coxautodata/waimak/dataflow/spark/SparkActions.scala

```diff
@@ -23,7 +23,7 @@ object SparkActions {
     */
   implicit class SparkDataFlowExtension(sparkDataFlow: SparkDataFlow) extends Logging {
 
-    type returnType = ActionResult[Dataset[_]]
+    type returnType = ActionResult
```

Comment:
Now ActionResult is defined in the package object, this is sort of a duplicate, maybe remove it?